src/components/Modals/SellCollectionModal/SellCollectionModal.container.ts

```diff
@@ -22,7 +22,7 @@ const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
 }
 
 const mapDispatch = (dispatch: MapDispatch): MapDispatchProps => ({
-  onSetMinters: (collection, minters) => dispatch(setCollectionMintersRequest(collection, minters))
+  onSetMinters: (collection, minters) => dispatch(setCollectionMintersRequest(collection, minters, false))
```

Comment:
Should this be changed via a FF or is it ok to remove the redirection on all cases?